examples/Demo/Shared/Pages/DataGrid/Examples/DataGridRemoteData.razor

```diff
@@ -4,12 +4,16 @@
 @inject NavigationManager NavManager
 
 <div style="height: 434px; overflow:auto;" tabindex="-1">
-    <FluentDataGrid ItemsProvider="foodRecallProvider" Virtualize="true" GenerateHeader="GenerateHeaderOption.Sticky" TGridItem="FoodRecall" >
+    <FluentDataGrid ItemsProvider="foodRecallProvider" Virtualize="true" ItemSize=46 GenerateHeader="GenerateHeaderOption.Sticky" TGridItem="FoodRecall" >
```

Comment:
Respecting naming conventions, could you add quotation marks to all attributes `ItemSize="46"`?